tests/installation/grub_test.pm

```diff
@@ -88,6 +88,8 @@ sub run {
         # avoid timeout for booting to HDD
         send_key 'ret';
     }
+    # Avoid return key not received occasionally for hyperv-uefi guest at first boot
+    send_key 'ret' if (check_var('HYPERV_VERSION', '2012r2') && check_var('UEFI', '1'));
```

Comment:
With one more time send_key, I am afraid the survival chance will not get significant improvement. What about changing line 89 to send_key_until_needlematch?